### ISSUE TITLE
Ripple Form Component Datepicker does not meet WCAG 2.0 Level AA: Acceptable compliance

### ISSUE THREAD

**Issue by haloabove** [dpc-sdp/ripple#1287]
**Describe the bug**
Ripple's From component Datepicker does not meet WCAG 2.0 Level AA: Acceptable compliance:
1. Not keyboard accessible
2. No action on keys: Enter, Space 

**To Reproduce**
Steps to reproduce the behaviour:
1. Navigate trough Date picker using keyboard only
2. use keyboard to select a date

**Expected behaviour**
1.The user should be able to select the date using only the keyboard.

**Actual behaviour**
1. No dates are selected when pressing the Enter or Space keys.

**Screenshots**

**Versions**
- Ripple: All

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome, Firefox
 - Version 111

**Smartphone (please complete the following information):**

**Additional context**
